Editor/Deployment/DeploymentSettings.cs

```diff
@@ -145,6 +145,8 @@ private set
         public bool IsCurrentStackModifiable =>
```

Comment:
This status seems to be meant for stacks that are "editable" - so i infer this means updatable, as opposed to deletable. Adding a new status for delete